app/controllers/pages_controller.rb

```diff
@@ -9,7 +9,7 @@ def about
     end
 
     def resources
-
```

Comment:
this is a redundant piece of code, it will automatically render a view with the name of the method. I'll approve this now because it wont break anything.